examples/controls_example/public/react_controls/data_controls/search_control/get_search_control_factory.tsx

```diff
@@ -148,7 +148,7 @@ export const getSearchControlFactory = ({
           if (currentDataView && currentField) {
             if (newSearchString) {
               api.setOutputFilter(
-                currentSearchTechnnique === 'match'
+                currentSearchTechnnique === 'match' || currentSearchTechnnique === undefined
```

Comment:
Maybe it's cleaner to do this in the comparators, like I did with options list?

Reply:
resolved in 116046f805dea370ca508da815169e2966332ee6